src/com/phronemophobic/membrane/term/main.clj

```diff
@@ -28,14 +29,15 @@ Replace membrane.term with your appropriate Clojure tools CLI launch sequence. F
 | clojure -M:membrane.term run-term -w 133 -h 60
 |")
 
-(defn- parse-pos-int [v]
-  (let [num (if (integer? v)
-                 v
-                 (when (and (string? v) (re-matches #"\d+" v))
-                   (Integer/parseInt v)))]
-    (if (and num (> num 0))
-      num
-      {:error "expected positive integer"})))
+(defn- parse-pos-int [min]
```

Comment:
Trying to think of a good name for this function now that it returns a parser and can accept non positive numbers depending on the input.

Reply:
Ah, interesting point. Maybe `pos-int-parser`?
Or `pos-int-parser-validator`?

Reply:
`pos?` is > 0 and it would be totally possible to pass a min that's less than 0. Is there a name that doesn't have `pos` in it? Maybe just `int-parser-validator`?

Reply:
Oh... right! `int-parser-validator` works for me!